mysql_env_cron.yml

```diff
@@ -0,0 +1,205 @@
+application:
+  configuration:
+    input.image: { ami: "us-east-1/ami-1ba18d72", user: "ec2-user" }
+    input.image-size: "m1.small"
+    #input.recipe-url: "http://qubell-starter-kit-artifacts.s3.amazonaws.com/qubell-bazaar/component-mysql-dev-cookbooks-training.tar.gz"
```

Comment:
I left these lines commented to see difference between env property and parameter.